resources/styles/components/breadcrumbs/step.less

```diff
@@ -0,0 +1,37 @@
+&-breadcrumbs-step {
+
```

Comment:
Those files are meant to be wrap imported only.  in the context of where it gets included, the full selector before `.openstax-breadcrumbs-step`.  See the end of this section in the [less docs](http://lesscss.org/features/#parent-selectors-feature).  This allows us to namespace those components without repeating the namespace in multiple files.